### PR TITLE
[WIP] Add Properties To Optimize The WASM binary Size per default

### PR DESCRIPTION
**Pull request by furesoft**
I've done a research and figured out a custom msbuild sdk would be for this purpose an overkill so I added the specific properties to the nuget package. I also added the RuntimeIdentifier to automaticly use the right runtime.

### CODE REVIEW

**Comment by furesoft** [comment]
One thing has to be done. The props file has to be included in the resulting nuget package with a nuspec file

**Comment by mhmd-azeez** [comment]
Helo @furesoft thank you very much for the PR, looks good from a first look. I will do some tests and merge it

**Comment by mhmd-azeez** [comment]
@furesoft I think if you put your changes into https://github.com/extism/dotnet-pdk/blob/main/src/Extism.Pdk/build/Extism.Pdk.targets It should work without needing a nuspec

**Comment by furesoft** [comment]
> @furesoft I think if you put your changes into https://github.com/extism/dotnet-pdk/blob/main/src/Extism.Pdk/build/Extism.Pdk.targets It should work without needing a nuspec

I did, so it's ready to be merged

File: src/Extism.Pdk/build/Extism.Pdk.targets [hunk 1]
```diff
@@ -1,6 +1,18 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<ItemGroup>
 		<TrimmerRootAssembly Include="Extism.Pdk" />
+
+		<RuntimeIdentifier>wasi-wasm</RuntimeIdentifier>
```
Comment (mhmd-azeez) [inline src/Extism.Pdk/build/Extism.Pdk.targets:5]: I think these have to be inside <PropertyGroup>

**Review by mhmd-azeez — CHANGES_REQUESTED** [review]

**Comment by mhmd-azeez** [comment]
@furesoft thank you very much for the PR

**Review by mhmd-azeez — APPROVED** [review]

**Pull request merged**